python/paddle/fluid/tests/unittests/test_collective_api_base.py

```diff
@@ -392,6 +392,14 @@ def check_with_place(
             need_result2 = [need_result[len(need_result) // 2 :]]
             self.assertEqual(need_result1, tr0_out)
             self.assertEqual(need_result2, tr1_out)
+        elif col_type == "gather":
+            # rank 0 gather all tensor
+            self.assertEqual(len(tr0_out), 2)
+            # rank 1 get nothing
+            self.assertEqual(len(tr1_out), 0)
+            # check values
+            np.testing.assert_allclose(input1, tr0_out[0], 1e-05)
```

Comment:
Assert equal instead of allclose. Same below.

Reply:
done